rust/cosmos_sdk_grpc_client/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "cosmos_sdk_grpc_client"
+description = "OKP4 CØSMOS gRPC client libraries based on Protobuf definitions."
+edition = "2021"
+license-file = "LICENSE"
+readme = "README.md"
```

Comment:
As we can specify a readme per target, I propose to have one for each target and specify here `rust/cosmos_sdk_grpc_client/README.md`